src/applications/vaos/actions/newAppointment.js

```diff
@@ -167,48 +182,55 @@ export function openFacilityPage(page, uiSchema, schema) {
     let facilities = null;
     let eligibilityData = null;
 
-    // If we have the VA systems in our state, we don't need to
-    // fetch them again
-    if (!systems) {
-      const userSystemIds = await getSystemIdentifiers();
-      systems = await getSystemDetails(userSystemIds);
-    }
-    const canShowFacilities =
-      newAppointment.data.vaSystem || systems?.length === 1;
-    const typeOfCareId = getTypeOfCare(newAppointment.data)?.id;
-
-    const hasExistingFacilities = !!newAppointment.facilities[
-      `${typeOfCareId}_${newAppointment.data.vaSystem}`
-    ];
-
-    if (canShowFacilities && !hasExistingFacilities) {
-      const systemId =
-        newAppointment.data.vaSystem || systems[0].institutionCode;
-      facilities = await getFacilitiesBySystemAndTypeOfCare(
-        systemId,
-        typeOfCareId,
-      );
-    }
+    try {
```

Comment:
I suggest reviewing this with whitespace changes off. A bunch of changes are just nesting differences.